pkg/version/update.go

```diff
@@ -0,0 +1,65 @@
+package version
+
+import (
+	"context"
+	"encoding/json"
+	"io"
+	"net/http"
+	"net/url"
+
+	"github.com/bacalhau-project/bacalhau/pkg/models"
+	"github.com/pkg/errors"
+)
+
+const endpoint = "http://update.bacalhau.org/version"
```

Comment:
Can we have a comment/todo to move this to configuration somewhere, otherwise we're at risk of doing a release just to satisfy a move to https or a diff domain?

Reply:
The reasons for this are:

* We do actually want everyone to always use this domain, and it shouldn't be configurable
* It's actually easier to migrate a hard-coded constant than a configuration value right now because we don't have any config migraiton capability

If the concern is about testing, there's a few ways we can do that even with a hard-coded value (e.g. a mock http client).